### PR TITLE
In reference to PR #902

### PR DESCRIPTION
**Pull request by Yuvraj-kadale**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

issue #563 Solved !!

### CODE REVIEW

**Comment by siddharth25pandey** [comment]
@Yuvraj-kadale PR #902 is already closed. 
Please create another PR & include all the details in one go
Thanks!

File: C/cp/Longest substring without repeating chars.cpp [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	if (max_len<len-j)
+	{
+		max_len=len-j;				//max_len=max(max_len,len-j);
+	}						
+	
+	return max_len-1;
+}
+
+int main(void)
+{
```
Comment (siddharth25pandey) [inline C/cp/Longest substring without repeating chars.cpp:54]: Take dynamic input

File: C/cp/Longest substring without repeating chars.cpp [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	char string[20]="dddddd";
+	int len=strlen(string);
+	printf ("Given Input Substring:"); 
+	printf("%s",string);				// length of the string
+	printf("\nlength of longest substring without repeating characters: %d\n", Lols(string,len));
+}
+
```
Comment (siddharth25pandey) [inline C/cp/Longest substring without repeating chars.cpp:61]: Add Sample Input & Output in the end of the code.

File: C/cp/Longest substring without repeating chars.cpp [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+Output: length of longest substring without repeating charecters: 3
+
+Input: Hackincodes
+Output: length of longest substring without repeating charecters: 7
+
+Input: ddddd
+Output: length of longest substring without repeating charecters: 1
+
+Input: 
+Output: length of longest substring without repeating charecters: 0  */
```
Comment (siddharth25pandey) [inline C/cp/Longest substring without repeating chars.cpp]: Shift this to the end

**Review by siddharth25pandey — COMMENTED** [review]
@Yuvraj-kadale Please do the suggested changes
Thanks!

**Comment by siddharth25pandey** [comment]
@Yuvraj-kadale Resolve the merge conflicts error

**Comment by Yuvraj-kadale** [comment]
> 
> 
> @Yuvraj-kadale Resolve the merge conflicts error

@siddharth25pandey Done!!

**Review by SKAUL05 — APPROVED** [review]

**Review by Kajol-Kumari — APPROVED** [review]
@Yuvraj-kadale maybe you should edit the tab setting of your editor as it seems like its taking more than 4 spaces for tab space :) Or you can use some code formatting tool to ease your formatting work instead.
Apart for this, code looks good to go :+1:

**Comment by Yuvraj-kadale** [comment]
> 
> 
> @Yuvraj-kadale maybe you should edit the tab setting of your editor as it seems like its taking more than 4 spaces for tab space :) Or you can use some code formatting tool to ease your formatting work instead.
> Apart for this, code looks good to go 👍

@Kajol-Kumari Thanks, I will look into my editors' settings

**Review by siddharth25pandey — APPROVED** [review]
@Yuvraj-kadale Good to go
Able to merge
Thanks!

**Pull request merged**